File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/Data/Database/MuscleGroupTestData.java
package com.jds.fitnessjunkiess.getfitapp.Data.Database;

import com.jds.fitnessjunkiess.getfitapp.Data.Entities.MuscleGroup;

import java.util.ArrayList;
import java.util.List;

public class MuscleGroupTestData {

  public static List<MuscleGroup> getData() {
    MuscleGroup m1 = new MuscleGroup();
    m1.setName("ABS");

    MuscleGroup m2 = new MuscleGroup();
    m2.setName("Back");

    MuscleGroup m3 = new MuscleGroup();
    m3.setName("Biceps");

    MuscleGroup m4 = new MuscleGroup();
    m4.setName("Legs");

    MuscleGroup m5 = new MuscleGroup();
    m5.setName("Triceps");

    MuscleGroup m6 = new MuscleGroup();
    m6.setName("Chest");

    List<MuscleGroup> muscleGroups = new ArrayList<>();
    muscleGroups.add(m1);
    muscleGroups.add(m2);
    muscleGroups.add(m3);
    muscleGroups.add(m4);
    muscleGroups.add(m5);
    muscleGroups.add(m6);

    return muscleGroups;
  }

}

File: app/src/main/java/com/jds/fitnessjunkiess/getfitapp/data/database/testData/MuscleGroupTestData.kt
package com.jds.fitnessjunkiess.getfitapp.data.database.testData

import com.jds.fitnessjunkiess.getfitapp.data.dataModels.MuscleGroup

import java.util.ArrayList

object MuscleGroupTestData {

    val data: List<MuscleGroup>
        get() {
            val m1 = MuscleGroup()
            m1.name = "ABS"

            val m2 = MuscleGroup()
            m2.name = "Back"

            val m3 = MuscleGroup()
            m3.name = "Biceps"

            val m4 = MuscleGroup()
            m4.name = "Legs"

            val m5 = MuscleGroup()
            m5.name = "Triceps"

            val m6 = MuscleGroup()
            m6.name = "Chest"

            val muscleGroups = ArrayList<MuscleGroup>()
            muscleGroups.add(m1)
            muscleGroups.add(m2)
            muscleGroups.add(m3)
            muscleGroups.add(m4)
            muscleGroups.add(m5)
            muscleGroups.add(m6)

            return muscleGroups
        }

}
